.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nvidia/cuda:12.6.2-cudnn-devel-ubuntu24.04
+
+RUN apt update -y && apt upgrade -y \
+  && apt install -y --no-install-recommends \
+    build-essential \
+    clang \
+    cmake \
+    gcc \
+    git \
+    libbenchmark-dev \
+    libgtest-dev
```

Comment:
I think we should clear the apt-cache to minimize layer size.

Reply:
Resolved in #4, thanks for the changes!